src/BuildScriptGeneratorCli/CliEnvironmentSettings.cs

```diff
@@ -17,6 +17,8 @@ public class CliEnvironmentSettings
         private const string DisableNodeJsEnvVarName = "DISABLE_NODEJS_BUILD";
         private const string EnableMultiPlatformBuildEnvVarName = "ENABLE_MULTIPLATFORM_BUILD";
         private const string GitHubActionsEnvVarName = "GITHUB_ACTIONS";
+        private const string GitHubActionsBuildContainerStartTimeEnvVarName = "GITHUB_BUILD_CONTAINER_START";
```

Comment:
nit: I'm not sure if we would reuse these environment variables in the future for tracking image pull times, but since we're currently only using it for GitHub Actions metrics, I would lean towards prefixing these environment variables with `GITHUB_ACTIONS`.

I would then call the two environment variables
- `GITHUB_ACTIONS_BUILD_START_TIME`
- `GITHUB_ACTIONS_BUILD_END_TIME`

Thoughts? CC: @kichalla @arroyc

Reply:
I think it's a good idea to have prefix👍